SU-HSDO/suhumsci
# SHS-5612:Implementation: Vertical Linked Card Title/Arrow Updates

# READY FOR REVIEW

## Summary
Update title and arrow in Vertical Linked Card

## Need Review By (Date)
05/03

## Urgency
medium

## Steps to Test
1. Go to the following page on Colorful and Traditional `/components/collections-cards`
2. Confirm the Vertical Linked Cards look now like the [Option A in Figma](https://www.figma.com/file/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?type=design&node-id=1009-112&mode=design&t=JlnFNF4nqEitHAn2-4)
3. Go to any page that has a view that is using Vertical Linked Card as pattern (I couldn't find any, so I went to `/mixed-pages/mixed-page-raised-cards` on both themes, edit any view to use that component
4. Confirm the same

## Notes
Font-sizes on both themes:

- Small viewports: 18px
- Large viewports: 20px

## Percy
https://percy.io/da667d23/suhumsci/builds/33953214

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
